docs/modules/nifi/pages/usage_guide/custom_processors.adoc

```diff
@@ -1,20 +1,21 @@
-= Loading Custom Components
+= Loading custom components
+:nifi-docs-custom-components: https://nifi.apache.org/docs/nifi-docs/html/developer-guide.html#introduction
 
-It is possible to develop https://nifi.apache.org/docs/nifi-docs/html/developer-guide.html#introduction[custom components] for Apache NiFi to extend the available functionality - most often this will probably be custom processors.
+It is possible to develop {nifi-docs-custom-components}[custom components] for Apache NiFi to extend the available functionality - most often this will probably be custom processors.
 
 For these to be available in the NiFi UI, they need to be present in the classpath at startup, so the nar bundles have to be injected into your NiFi instance.
 
 The Stackable Data Platform supports two ways to achive this goal, both of which are described below.
 
 Please note that building your own Docker image is the recommended solution, as giving multiple pods access to a shared PVC can often be tricky (see comment on access mode in the second section).
 
-== Custom Docker Image
+== Custom Docker image
 
-You can extend the official Stackable NiFi image and simply copy all required nar files into the classpath of Nifi.
-The benefit of this method is that there is no need for any config overrides or extra mounts, you can simply use any ouf our NiFi examples, swap the image and your components will be available.
+You can extend the official Stackable NiFi image and copy all required nar files into the classpath of Nifi.
```

Comment:
TIL: NAR files are a thing. I assumed it was a typo for JAR.

Maybe it should be emphasised in some way?

```suggestion
You can extend the official Stackable NiFi image and copy all required _nar_ files into the classpath of Nifi.
```